src/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -26,6 +27,11 @@ private static bool UseManagedHandler
                     return true;
                 }
 
+                if (AppContext.TryGetSwitch(AppCtxManagedHandlerSettingName, out bool isManagedEnabled) && isManagedEnabled)
```

Comment:
This is checking the environment variable and then checking the appconfig switch.  As @eerhardt mentions at https://github.com/dotnet/corefx/pull/26403/files#r162389382, that globalization check does it in reverse; we should be consistent.

Reply:
I can change it. But there is no functional impact, right? This is OR so we will us handler if either value is set.

Reply:
I agree with @stephentoub. 

> But there is no functional impact, right? 

yes, there is a functional impact. imagine someone set the value to false in the config file and set it true in the environment variable. the app configs should have a higher priority than the environment variable.

Reply:
Also to mention, you have to get the logic right. AppContext.TryGetSwitch returns true if the switch defined in the config file even if the switch value is false. so you would just move the environment variable reading to the else block of AppContext.TryGetSwitch check.